packages/runtime/container-runtime/src/test/opLifecycle/opDecompressor.spec.ts

```diff
@@ -19,7 +20,7 @@ interface ITestMessageContents {
 }
 
 function generateCompressedBatchMessage(length: number): ISequencedDocumentMessage {
-	const batch: ContainerRuntimeMessage[] = [];
+	const batch: TypedContainerRuntimeMessage<ContainerMessageType.FluidDataStoreOp, string>[] = [];
```

Comment:
Ummm what why isn't it `IEnvelope`  -_-

Reply:
I guess it just needs something and doesn't care about the actual message format.  Maybe just put a comment saying "this isn't what any real message looks like, it's just for testing" or something.

Reply:
Feels like TypedContainerRuntimeMessage shouldn't be exported from messageTypes if it can be helped.
Definitely need to call out things that aren't real.